test/openshift/e2e/parallel/1-031_validate_toolchain/01-check.yaml

```diff
@@ -8,7 +8,7 @@ commands:
       # These variables need to be maintained according to the component matrix: https://spaces.redhat.com/display/GITOPS/GitOps+Component+Matrix
       expected_kustomizeVersion='v5.4.2'
       expected_helmVersion='v3.15.2'
-      expected_argocdVersion='v2.12.6+4dab5bd'
+      expected_argocdVersion='v2.12.7'
```

Comment:
There should be a short commit id too added isn't it ?

Reply:
This needs to be matched to the version in `go.mod` which is automatically updated when running `go mod download && go mod tidy && go mod vendor`

Reply:
I think the test will fail as it will try to match `v2.12.7`

Reply:
The test seems to be succeeding. The version in `go.mod` is also reflecting `v2.12.7` matching the value in test.